.markdownlint.json

```
@@ -0,0 +1,3 @@
+{
+	"extends": "packages/scripts/config/.markdownlint.json"
```

```suggestion
	"extends": "@wordpress/scripts/config/.markdownlint.json"
```

Would it work if we reference it as a package rather than a local file like in other config files?

---

No unfortunately not, it requires a path and not package